conversion/conversion.go

```diff
@@ -329,7 +327,7 @@ func populateDataConv(conv *internal.Conv, config writer.BatchWriterConfig, clie
 		migrationData := metrics.GetMigrationData(conv, "", "", constants.DataConv)
 		serializedMigrationData, _ := proto.Marshal(migrationData)
 		migrationMetadataValue := base64.StdEncoding.EncodeToString(serializedMigrationData)
-		_, err := client.Apply(metadata.AppendToOutgoingContext(context.Background(), migrationMetadataKey, migrationMetadataValue), m)
+		_, err := client.Apply(metadata.AppendToOutgoingContext(context.Background(), constants.MigrationMetadataKey, migrationMetadataValue), m)
```

Comment:
Is this change backward compatible? Can it cause a split with the existing migration usage data that we have/and or break the migration dashboard we have created?